siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -1249,8 +1260,8 @@ def _check_configs(self):
         for i, dev in enumerate(self._llrfs):
             if dev.connected:
                 value = _updt_bit(value, 2*i, 0)
-                fim_orbit = dev.fast_interlock_monitor_orbit
-                fim_manual = dev.fast_interlock_monitor_manual
+                fim_orbit = dev['FIMLLRF1-Sts']
+                fim_manual = dev['FIMManual-Sts']
```

Comment:
Shouldn't we keep using the property for manual, since that's not going to change depending on the LLRF? It's not a physical connection, after all.

Reply:
Ok, done